python/setup.cfg

```diff
@@ -48,3 +49,12 @@ skip=
     build
     dist
     __init__.py
+
+
+[options]
+packages = find:
+install_requires =
+    numpy
+    numba>=0.49
+    Cython>=0.29,<0.30
```

Comment:
I think we can remove `Cython` from here since it's only needed by the build system but not for actual install.

Reply:
Done